devel/re-flex/Portfile

```diff
@@ -26,8 +26,9 @@ long_description    RE/flex is a regex-centric, fast lexical analyzer generator
 
 homepage            https://www.genivia.com/doc/reflex/html/
 
-# identical executable name
-conflicts           reflex
+# rename to avoid conflict (identical executable name) with pre-existing
+# "reflex" port, using the expected name given this port's name.
```

Comment:
this is a comment in the Portfile that will never been seen by the end-user. As suggested, please add "notes" so that a message will be displayed to the user when the port is installed.